MarkBind/markbind
# Implement Disqus plugin

**What is the purpose of this pull request?**


This PR resolves #481. 



**Overview of changes:**
Setup Disqus plugin with lazy loading using IntersectionObserver API. 

**Anything you'd like to highlight / discuss:**
- Code quality / implementation
- Documentation (whether the explanation is clear enough)

**Testing instructions:**
Add `<disqus/>` to any page and `markbind serve`. 

You should be able to see the disqus comments section (it should load only when you scroll near it). 



**Proposed commit message: (wrap lines at 72 characters)**
Implement Disqus plugin.

Users may want to allow readers to comment on content in the website.
This can be an important feature for educational websites as it enables
better student interaction.

Let's integrate Disqus into MarkBind to enable this commenting feature. 



---

**Checklist:** :ballot_box_with_check:
